liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/AbstractLibertySupport.java

```diff
@@ -298,6 +299,15 @@ protected ArtifactItem createArtifactItem(String groupId, String artifactId, Str
                 }
             }
 
+            // Remove test scoped dependencies
+            Set<Artifact> dependenciesWithoutTest = new HashSet<Artifact>();
+            for (Artifact artifact : resolvedDependencies) {
+                if (!"test".equals(artifact.getScope())) {
+                    dependenciesWithoutTest.add(artifact);
+                }
+            }
+            resolvedDependencies = dependenciesWithoutTest;
+
```

Comment:
I think this is correct, and the tests look good... but it'd be a bit quieter if we filtered out the test scope artifacts before finding transitive dependencies.

E.g for this one:
```xml
		<dependency>
			<groupId>org.slf4j</groupId>
			<artifactId>slf4j-log4j12</artifactId>
			<version>1.7.30</version>
			<scope>test</scope>
		</dependency>
```

I get all this output, but the two WARNING(s) are probably enough:

> 
> [INFO] Adding transitive dependency with scope: test and GAV: org.slf4j:slf4j-api:1.7.30
> [INFO] Adding transitive dependency with scope: test and GAV: log4j:log4j:1.2.17
> [WARNING] Unable to find artifact matching groupId org.slf4j, artifactId slf4j-log4j12, and version null in either project dependencies or in project dependencyManagement.
> [WARNING] copyDependencies failed for dependency with groupId org.slf4j, artifactId slf4j-log4j12 and type jar. No matching resolved dependencies were found.

Reply:
I sent a PR to your PR branch.  I haven't tested it yet though.

Reply:
Thanks, I tested the changes out, it looks good.